### PR TITLE
Check for theme params on the URL

### PR DESCRIPTION
**Pull request by talvasconcelos**
- Fixed an inconsistency on the apply gradient, where the background color wouldn't take effect until refresh

- added to base.html a snippet to look for URL params related to theme
- allow for enforcing the desired styling for display pages (tpos, events, tickets, etc...)

### CODE REVIEW

File: lnbits/templates/base.html [hunk 1]
```diff
@@ -258,6 +258,62 @@
         { value: 'fi', label: 'Suomi', display: '🇫🇮 FI' }
       ]
     </script>
+    <script>
+      const currentUrl = window.location.href
+      let url = new URL(currentUrl)
```
Comment (motorina0) [inline lnbits/templates/base.html:263]: why not `const`?

File: lnbits/templates/base.html [hunk 2]
```diff
@@ -258,6 +258,62 @@
...
+      let params = new URLSearchParams(url.search)
+      let q = Quasar.plugins.LocalStorage
+
+      const setColors = () => {
```
Comment (motorina0) [inline lnbits/templates/base.html:267]: why define a separate internal function if only called once?

Comment (arcbtc) [reply lnbits/templates/base.html:267]: isnt it neater?

Comment (dni) [inline lnbits/templates/base.html:267]: i think this should be on `created()` in base.js not in the html, i would only put variables there

Comment (talvasconcelos) [reply lnbits/templates/base.html:267]: it was already set this way... i can change it, but looks neater as @arcbtc said!

does it have memory/performance impact?

**Review by motorina0 — APPROVED** [review]

**Pull request merged**